src/WebJobs.Extensions.DurableTask/Options/NotificationOptions.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Options
+{
+    /// <summary>
+    /// Configuration of the notification options
+    /// for the Durable Task Extension.
+    /// </summary>
+    public class NotificationOptions
+    {
+        /// <summary>
+        /// Gets or sets the base URL for the HTTP APIs managed by this extension.
+        /// </summary>
+        /// <remarks>
+        /// This property is intended for use only by runtime hosts.
+        /// </remarks>
+        /// <value>
+        /// A URL pointing to the hosted function app that responds to status polling requests.
+        /// </value>
+        public Uri ApiUrl { get; set; }
```

Comment:
Don't think that this technically should be in NotificationOptions, as this is more used for our API endpoints.

Reply:
Reverted it back to `NotificationUrl` in `DurableTaskOptions.cs`